src/PaymentGateways/Gateways/PayPalCommerce/payPalCommerceGateway.tsx

```diff
@@ -134,15 +100,7 @@ import createSubscriptionPlan from './resources/js/createSubscriptionPlan';
 
         formData.append('give_payment_mode', 'paypal-commerce');
 
-        const eventTicketsTotalAmount = eventTickets ? getEventTicketsTotalAmount(JSON.parse(eventTickets)) : 0;
-        const isSubscription = subscriptionPeriod ? subscriptionPeriod !== 'one-time' : false;
-        if (!isSubscription) {
-            formData.append('give-amount', getAmount() + eventTicketsTotalAmount);
-        } else {
-            formData.append('give-amount', getAmount()); // We don't want to charge the event tickets for each subscription renewal
-        }
-
-        formData.append('give-event-tickets-total-amount', String(eventTicketsTotalAmount));
+        formData.append('give-amount', amount.toString());
```

Comment:
Sooo, I just discovered this issue using decimals. My form had fee recovery and event tickets and the calculated amount was this really long number `151.80226570546` and apparently PayPal has a rule for decimals that depend on currency.  It looks like we were previously able to get around this by doing a little math calculation so I guess we should stick with what was working before.

```
{
    "response": "{\"name\":\"UNPROCESSABLE_ENTITY\",\"details\":[{\"field\":\"/purchase_units/@reference_id=='default'/amount/value\",\"value\":\"151.80226570546\",\"issue\":\"DECIMAL_PRECISION\",\"description\":\"If the currency supports decimals, only two decimal place precision is supported.\"}],\"message\":\"The requested action could not be performed, semantically incorrect, or failed business validation.\",\"debug_id\":\"d6346ee6a5cf0\",\"links\":[{\"href\":\"https://developer.paypal.com/api/rest/reference/orders/v2/errors/#DECIMAL_PRECISION\",\"rel\":\"information_link\",\"method\":\"GET\"}]}"
}
```

```suggestion
        formData.append('give-amount', (Math.round(amount * 100) / 100).toString());
```

Reply:
@jonwaldstein Nice catch! But I'm wondering if this is something that the `useFormaData` hook should be dealing with. I think it's a good idea to always return this value with fewer decimals since this is the most common use case for the amount field.

Reply:
@glaubersilva yeah i'm thinking the same.  Most gateways will end up using the `amountInMinorUnits` but there are some exceptions like PayPal that need the float amount and I think doing this little math conversion will help normalize the value for those situations and does not force a certain amount of decimals.

Reply:
Resolved in d745ac413287217567ef6ec84f505de0433b2f72

Reply:
@jonwaldstein This is ready for re-review.